Build failure: openvdb (dependency of bambu-studio, incompatible boost versions)

### Steps To Reproduce

Steps to reproduce the behavior:
1. build openvdb

### Build log

I think that the following lines is representative enough.

```
       last 25 log lines:
       > -- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
       > -- Configuring for OpenVDB Version 12.0.0
       > -- Configuring for OpenVDB ABI Version 12
       > -- ----------------------------------------------------
       > -- ------------- Configuring OpenVDBCore --------------
       > -- ----------------------------------------------------
       > CMake Warning (dev) at openvdb/openvdb/CMakeLists.txt:112 (find_package):
       >   Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
       >   --help-policy CMP0167" for policy details.  Use the cmake_policy command to
       >   set the policy and suppress this warning.
       >
       > This warning is for project developers.  Use -Wno-dev to suppress it.
       > 
       > CMake Error at /nix/store/irwcgpm8csj7br49jih6kdpqkaqya3vf-cmake-3.30.4/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
       >   Could NOT find Boost: Found unsuitable version "1.79.0", but required is at
       >   least "1.80" (found
       >   /nix/store/p0a0h9zwmgqxn1vhl7ang0zxrrl1x60j-boost-1.79.0-dev/include, found
       >   components: iostreams regex)
       > Call Stack (most recent call first):
       >   /nix/store/irwcgpm8csj7br49jih6kdpqkaqya3vf-cmake-3.30.4/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:601 (_FPHSA_FAILURE_MESSAGE)
       >   /nix/store/irwcgpm8csj7br49jih6kdpqkaqya3vf-cmake-3.30.4/share/cmake-3.30/Modules/FindBoost.cmake:2409 (find_package_handle_standard_args)
       >   openvdb/openvdb/CMakeLists.txt:112 (find_package)
       >
       > 
       > -- Configuring incomplete, errors occurred!
       For full logs, run 'nix log /nix/store/fvig2bd6hfrh8njaplb84ia2f18h7mf7-openvdb-12.0.0.drv'.
```

### Additional context

Add any other context about the problem here.

### Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@guibou

### Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.58, NixOS, 24.11 (Vicuna), 24.11.20241030.d565bce`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.9`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/9aq8y2y2v47626vy3nm5wmy5x2a39bdp-source`

---

--- a/pkgs/applications/misc/bambu-studio/default.nix
+++ b/pkgs/applications/misc/bambu-studio/default.nix
@@ -1,9 +1,6 @@
 {
   stdenv,
   lib,
-  openexr,
-  jemalloc,
-  c-blosc,
   binutils,
   fetchFromGitHub,
   cmake,
@@ -55,17 +52,6 @@ let
           "--enable-debug=no"
         ];
       });
-
-  openvdb' = openvdb.overrideAttrs (old: {
-    buildInputs = [
-      openexr
-      boost179
-      tbb_2021_11
-      jemalloc
-      c-blosc
-      ilmbase
-    ];
-  });
 in
 stdenv.mkDerivation rec {
   pname = "bambu-studio";
@@ -111,7 +97,7 @@ stdenv.mkDerivation rec {
     mpfr
     nlopt
     opencascade-occt_7_6
-    openvdb'
+    openvdb
     pcre
     tbb_2021_11
     webkitgtk_4_0

